frontend/src/screens/TestScreen/TestScreen.tsx

```diff
@@ -0,0 +1,199 @@
+import React, { useEffect, useState } from "react";
+import { View, Text, StyleSheet, TouchableOpacity } from "react-native";
+import FirebaseInteractor from "../../firebase/firebaseInteractor";
+import { DraxProvider } from "react-native-drax";
+import { useNavigation } from "@react-navigation/core";
+import AnimatedBar from "react-native-animated-bar";
+
+const fi = new FirebaseInteractor();
+
+const TEST_HEADER_OPTIONS = {
+    headerTitle: () => { return <Text style={{color:"#5eafdf", fontSize:20}}>Test Your Knowledge</Text> },
+    title: '',
+    headerTitleAlign: "center" as "center",
+    headerShadowVisible: false,
+    headerTintColor: '#FFF',
+    headerBackTitle: '',
+    headerStyle: {
+      backgroundColor: '#FFF',
+      elevation: 0,
+      shadowOpacity: 0,
+      borderBottomWidth: 0,
+    }
+  }
+
+export interface TestScreenProps {
+    route: any;
+}
+
+export default function TestScreen(props: TestScreenProps) {
+
+    const navigation = useNavigation();
+    const { testId } = props.route.params;
+    const [page, setPage] = useState<number>(1);
+    const numQuestions = 50;
+    const TESTWORDS = require('./testWords.json');
+    const [answer, setAnswer] = useState<string>("");
+    const [answers, setAnswers] = useState<string[]>([]);
+    const answerKey = ['no', 'yes', 'no', 'no', 'yes', 'yes', 'no', 'yes', 'yes', 'no', 'yes', 'no', 'yes', 'no', 'yes', 'yes', 'yes', 'no', 'yes', 'yes', 'yes', 'no', 'no', 'yes', 'yes', 'yes', 'no', 'yes', 'no', 'yes', 'no', 'no', 'yes', 'no', 'yes', 'yes', 'no', 'yes', 'no', 'yes', 'no', 'yes', 'no', 'yes', 'no', 'yes', 'yes', 'yes', 'yes', 'yes'];
```

Comment:
This is a little nit-picky but could we split long lines like these into multiple lines for readability? I feel like this could apply to the `TouchableOpacity`s in lines 110-117, and it might be worth double checking other files as well.

Reply:
I'd also be interested in hearing your thoughts on possibly moving this answer key out of the code - maybe it would be appropriate to have it in our representation of the test word pairs in Firestore?